Mantle/NSValueTransformer+MTLPredefinedTransformerAdditions.h

```diff
@@ -10,6 +10,14 @@
 
 #import "MTLTransformerErrorHandling.h"
 
+#define DEPRECATED_BUT_UNAVAILABLE_IN_SWIFT(...) __attribute__((availability(macosx,deprecated=10.0,message="" __VA_ARGS__))) \
```

Comment:
Seen this three times now — I guess that means this needs to go in an `MTLDefines`?